### ISSUE TITLE
Provide an option to change the path of a local state file

### ISSUE THREAD

**Issue by blake-mealey** [blake-mealey/mantle#113]
If someone has 2 mantle project files in the same directory, they cannot both use `state: local` config without overriding each other and causing weird behaviour. Additionally, this may also be helpful regarding #112. Could be implemented with a new config:

```yml
state:
  local: path/to/state.mantle-state.yml
```

Which would make for three possible state configs:

```yml
# local (default)
state: local

# local (custom file path)
state:
  local: path/to/state.mantle-state.yml

# remote
state:
  remote:
    region: [us-west-2]
    bucket: mantle-states
    key: state
```

**Comment by blake-mealey** [comment]
This has been implemented in #130 but using the format:

```yml
state:
  localCustom: path/to/state.mantle-state.yml
```

I would like to use the format described above but I need to figure out how to make serde use this format first.

**Comment by blake-mealey** [comment]
I modified the format. See #131 for details
